src/main/resources/application.yml

```diff
@@ -3,7 +3,9 @@ spring:
     default-property-inclusion: non_null
     date-format: "yyyy-MM-dd'T'HH:mm:ss.SSS'Z'"
   data.mongodb.uri: mongodb://${MONGO_USERNAME}:${MONGO_PASSWORD}@mongodb:27017/termsDB?authSource=admin&authMechanism=SCRAM-SHA-1
-  security.oauth2.resourceserver.jwt.jwk-set-uri: ${JWK_KEY_SET_URI:http://sso:8084/auth/realms/fdk/protocol/openid-connect/certs}
+  security.oauth2.resourceserver.jwt:
+    jwk-set-uri: ${JWK_KEY_SET_URI:http://sso:8084/auth/realms/fdk/protocol/openid-connect/certs}
+    issuer-uri: ${SSO_HOST:https://sso.staging.fellesdatakatalog.digdir.no}/auth/realms/fdk
```

Comment:
Det er greit å gjenbruke SSO_HOST for begge to

Reply:
Åh fan, glemte av dette. Har jo ikke lagt inn `SSO_HOST` i helm, dette ville feilet på en ganske mystifistisk måte i prod :sweat_smile:

Reply:
fixn'd'it